components/tools/OmeroPy/src/omero/plugins/prefs.py

```diff
@@ -150,7 +150,12 @@ def _configure(self, parser):
             "KEY", nargs="*", help="Names of keys in the current profile")
         get.add_argument(
             "--hide-password", action="store_true",
-            help="Hide values of password keys in the current profile")
+            help="[Deprecated; this is the default now] Hide values of "
```

Comment:
You can also use `SUPPRESS` if that's preferred: https://github.com/openmicroscopy/openmicroscopy/blob/v5.4.0/components/tools/OmeroPy/src/omero/plugins/import.py#L325